From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bradley Dice <bdice@bradleydice.com>
Date: Wed, 29 Jan 2025 12:00:00 -0800
Subject: [PATCH 1/2] Use standard CMake FindCUDAToolkit module

The custom FindCUDAToolkit.cmake does not work with conda-forge's
CUDA packages. Use the standard CMake module instead.

---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,8 +5,6 @@
 cmake_minimum_required(VERSION 3.18)
 project(cuda-tile-python)
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
-
 if (MSVC)
     set(CMAKE_CXX_STANDARD 20)
 else()
-- 
2.48.1

